ton_client/src/net/mod.rs

```diff
@@ -132,6 +133,22 @@ impl Default for NetworkConfig {
     }
 }
 
+#[derive(Serialize, Deserialize, ApiType, Clone)]
+pub struct ParamsOfQuery {
+    /// GraphQL query text.
+    pub query: String,
+    /// Variables used in query.
+    pub variables: Option<serde_json::Value>,
+    /// Query timeout in ms.
```

Comment:
Describe function behaviour when timeout is set or not set. There are two different fucntions in one